### PR TITLE
zh-CN: update glossary/cookie

### PR DESCRIPTION
**Pull request by PassionPenguin**
#### Modification

- improvements on the first way to set/modify cookie

### CODE REVIEW

File: files/zh-cn/glossary/cookie/index.md [hunk 1]
```diff
@@ -9,7 +11,7 @@ Cookie 就是访问者在访问网站后留下的一个信息片段。
 
 Cookie 用于个性化用户的体验。可能包含用户在访问网站时的参数或输入。用户可以自定义浏览器来接受，拒绝或删除 Cookie。
 
-`Cookie 可以在服务器级别设置和修改，使用 Set-Cookie` [HTTP header](/zh-CN/docs/Web/HTTP/Cookies), 或者在 JavaScript 中用 [`document.cookie`](/zh-CN/docs/Web/API/Document/cookie).
+Cookie 可以通过在服务器中使用 `Set-Cookie` [HTTP header](/zh-CN/docs/Web/HTTP/Cookies), 或者在 JavaScript 中用 [`document.cookie`](/zh-CN/docs/Web/API/Document/cookie) 进行设置、修改。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/cookie/index.md:14]:
```suggestion
Cookie 可以通过在服务器中使用 `Set-Cookie` [HTTP 标头](/zh-CN/docs/Web/HTTP/Cookies)，或者在 JavaScript 中用 [`document.cookie`](/zh-CN/docs/Web/API/Document/cookie) 进行设置、修改。
```

File: files/zh-cn/glossary/cookie/index.md [hunk 2]
```diff
@@ -9,7 +11,7 @@ Cookie 就是访问者在访问网站后留下的一个信息片段。
 
 Cookie 用于个性化用户的体验。可能包含用户在访问网站时的参数或输入。用户可以自定义浏览器来接受，拒绝或删除 Cookie。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/cookie/index.md:12]:
```suggestion
Cookie 用于个性化用户访问网站的体验。它可能包含用户在访问网站时的偏好或输入。用户可以自定义浏览器来接受、拒绝或删除 Cookie。
```

File: files/zh-cn/glossary/cookie/index.md [hunk 3]
```diff
@@ -9,7 +11,7 @@ Cookie 就是访问者在访问网站后留下的一个信息片段。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/cookie/index.md:11]: L10：

```md
cookie 是网站通过 web 浏览器留在访问者电脑上的一小段信息。
```

Comment (PassionPenguin) [reply files/zh-cn/glossary/cookie/index.md:11]: oops, this is changed...!

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**